tests/signer-authorization.ts

```diff
@@ -30,35 +22,42 @@ describe("signer-authorization", () => {
       wallet.publicKey,
       null,
       0
-    )
+    );
 
     withdrawDestinationFake = await spl.createAccount(
       connection,
       wallet.payer,
       mint,
       walletFake.publicKey
-    )
+    );
+
+    const airdropSignature = await connection.requestAirdrop(
+      walletFake.publicKey,
+      1 * anchor.web3.LAMPORTS_PER_SOL
+    );
+
+    const latestBlockHash = await connection.getLatestBlockhash();
 
     await connection.confirmTransaction(
-      await connection.requestAirdrop(
-        walletFake.publicKey,
-        1 * anchor.web3.LAMPORTS_PER_SOL
-      ),
+      {
+        blockhash: latestBlockHash.blockhash,
+        lastValidBlockHeight: latestBlockHash.lastValidBlockHeight,
+        signature: airdropSignature,
+      },
       "confirmed"
-    )
-  })
+    );
+  });
 
   it("Initialize Vault", async () => {
```

Comment:
This is a nitpick, and was an issue in the old code, but this is a mix of TDD/BDD style.

Either

```
describe('apple')
it('is delicious)

describe('apples')
test('the apple tastes delicious)
```

You might want to use `test()`